--- origsrc/gnopernicus-1.0.5/configure.in	2006-05-23 02:29:38.000000000 -0500
+++ src/gnopernicus-1.0.5/configure.in	2006-06-09 12:33:21.281250000 -0500
@@ -150,6 +150,12 @@
 AC_CHECK_LIB(popt, poptDupArgv,,
 AC_MSG_ERROR([You must have popt 1.5 or greater to compile the utilities.]))
 
+dnl some OS's have 'res_init' in libc, others like Cygwin use -lresolv,
+dnl and Darwin also has some symbol mangling which necessitates a different check
+dnl (hence the second line, based on ORBit2's configure.in).
+
+AC_CHECK_FUNC(res_init,,[AC_CHECK_LIB(resolv,res_init)])
+AC_CHECK_LIB(resolv,res_9_init)
 
 dnl ***************************************************************************
 dnl *               deal with diffrent OSs for BRAILLE (serial port)          *
